pkg/binutils/custom.go

```diff
@@ -9,3 +9,8 @@ func SetupCustomBin(app *application.Avalanche, subnetName string) string {
 	// Just need to get the path of the vm
 	return app.GetCustomVMPath(subnetName)
 }
+
+func SetupAPMBin(app *application.Avalanche, vmid string) string {
```

Comment:
why is this in custom? probably this can be named utils or something else to avoid confusion with custom vm stuff

Reply:
I thought it belonged there because it has the same structure as a custom. Instead of loading a binary from our "package management" directories, it just copies the VM over from a special folder.